openpathsampling/experimental/path_tree/svg.py

```diff
@@ -0,0 +1,82 @@
+import xml.etree.ElementTree as ET
+from .options import canonicalize_mover
+from .plotter import PathTreePlotter
+
+def _stringify_values(dct):
+    # also known as "XML is stupid"
+    return {k: str(v) for k, v in dct.items()}
+
+class SVGLines(PathTreePlotter):
+    def __init__(self, options=None, hscale=10, vscale=10):
+        super().__init__(options)
+        self.hscale = hscale
+        self.vscale = vscale
+        self._reset()
+
+    def _reset(self):
+        self.trajectories = []
+        self.connectors = []
+        self.min_x = float("inf")
+        self.max_x = float("-inf")
+
+    # @staticmethod
+    # def step_basics(step, options):
+    #     mover = canonicalize_mover(step.mover)
+    #     mover_options = options.movers[mover]
+    #     color = mover_options.color
+    #     plot_segments = mover_options.get_left_right(step)
+    #     return mover, color, plot_segments
```

Comment:
leftover debug code?

Reply:
refactored into a superclass, actually (same difference!) Removed in [16ede3b](https://github.com/openpathsampling/openpathsampling/pull/1048/commits/16ede3b5a486af0fff482dbb7f92382a90f23f57)